en/Import notes/Import Textbundle files.md

```diff
@@ -0,0 +1,19 @@
+---
+permalink: import/textbundle
+---
+Obsidian allows you to easily migrate your notes from apps that support the [Textbundle](https://textbundle.org/) format using the [[Importer|Importer plugin]]. 
+
+The Textbundle format bundles Markdown text and all referenced images into a single file, providing a more seamless way to move out of a sandboxed application. Textbundle export is supported by [many apps](https://textbundle.org/), including Agenda, Craft, Taio, Ulysses, Zettlr, and more.
+
+## Import Textbundle files into Obsidian
+
+You will need the official Obsidian [[Importer]] plugin, which you can [install here](obsidian://show-plugin?id=obsidian-importer).
+
+1. Open **Settings**.
+2. Go to **Community Plugins** and [install Importer](obsidian://show-plugin?id=obsidian-importer).
+3. Enable the Importer plugin.
+4. Open the **Importer** plugin using the command palette or ribbon icon.
+5. Under **File format** choose **Textbundle (.textbundle).**
+6. Select the location of your Texbundle files.
```

Comment:
Unfortunately right now it's implemented in a way where you can only import one textbundle file at a time. Not ideal, and probably could be solved with some time if we decide it's important.

But in the mean time we might want to update this phrasing accordingly.